tests/std/tests/P0660R10_jthread_and_cv_any/test.cpp

```diff
@@ -166,7 +166,7 @@ int main() {
             mutex m;
             condition_variable_any cv;
             unique_lock lck{m};
-            assert(cv.wait_for(lck, move(token), chrono::steady_clock::duration::max(), [] { return false; }) == false);
+            assert(cv.wait_for(lck, move(token), infinity, [] { return false; }) == false);

```

Comment:
This looks like it won't compile - `infinity` is a `time_point`, not a `duration`. Occurs below.